katalog/grafana/deploy.yml

```diff
@@ -56,21 +106,96 @@ spec:
         - mountPath: /grafana-dashboard-definitions/0/goldpinger
           name: grafana-dashboard-definitions-goldpinger
           readOnly: false
```

Comment:
Isn't this redundant?

Reply:
No

Reply:
Sorry, I was just referring to the `readOnly: false` part. 😅

Reply:
> readOnly: An optional boolean value indicating whether the volume is to be “ControllerPublished” (attached) as read only. Default is false. This value is passed to the CSI driver via the readonly field in the ControllerPublishVolumeRequest.

Yes, it is!

Reply:
Fixed in latest commit ;)